cmd/basin/publication.go

```diff
@@ -48,6 +48,7 @@ func newPublicationCommand() *cli.Command {
 
 func newPublicationCreateCommand() *cli.Command {
 	var owner, dburi, provider string
+	var secure bool
```

Comment:
we're adding a secure flag to every command where the default is true, we need a flag for local testing where we have no tls connection